pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -152,13 +153,16 @@ func validateJobCreate(job *v1alpha1.Job, reviewResponse *v1beta1.AdmissionRespo
 		}
 
 		if err := validatePolicies(task.Policies, field.NewPath("spec.tasks.policies")); err != nil {
-			msg = msg + err.Error() + fmt.Sprintf(" valid events are %v, valid actions are %v",
+			msg += err.Error() + fmt.Sprintf(" valid events are %v, valid actions are %v",
 				getValidEvents(), getValidActions())
 		}
-
+		podName := jobhelpers.MakePodName(job.Name, task.Template.Name, index)
+		msg += validateK8sPodNameLength(podName)
 		msg += validateTaskTemplate(task, job, index)
 	}
 
+	msg += validateK8sSvcNameLength(job)
```

Comment:
I am confused about the func name "validateK8sSvcNameLength".  No services here treatment in func, "validateJobName" seems better.

Reply:
Because we use job name as pod svc name .

Reply:
Ok,i have change the method name.